pkg/database/connection.go

```diff
@@ -70,10 +70,21 @@ func (dao *DAO) InitializeTables() {
 	_, err = dao.pool.Exec(context.Background(), "DROP TABLE search.edges")
 	checkError(err, "Error dropping table search.resources.")
 
-	_, err = dao.pool.Exec(context.Background(), "CREATE TABLE IF NOT EXISTS search.resources (uid TEXT PRIMARY KEY, cluster TEXT, data JSONB)")
+	_, err = dao.pool.Exec(context.Background(), "CREATE TABLE IF NOT EXISTS search.resources (uid TEXT PRIMARY KEY, cluster TEXT, data JSONB, kind TEXT, namespace TEXT, name TEXT)")
```

Comment:
I thought we wouldn't need this change and we would use the jsonb GIN index instead.

Reply:
Yes, we are removing this just wanted to keep incase we wanted to benchmark more queries.

Reply:
(same goes for the model Resources PrimaryProperties field)